mods/d2k/rules/defaults.yaml

```diff
@@ -457,18 +457,20 @@
 	Inherits: ^Building
 	Inherits@selection: ^SelectableCombatBuilding
 	WithSpriteTurret:
+		RequiresCondition: !build-incomplete
 	AttackTurreted:
 	RenderRangeCircle:
 	DetectCloaked:
 		Range: 1c768
 	-GivesBuildableArea:
-	-WithMakeAnimation:
-	-WithCrumbleOverlay:
 	-Capturable:
-	-WithSpriteBody:
+	WithSpriteBody:
+		PauseOnCondition: build-incomplete
```

Comment:
Pausing does not hide the sprite, just stops any idle animation. You can tell that this is rendered twice by the shadow being darker than it should be.

If this is only for the building placement preview you can instead use `SequencePlaceBuildingPreview` like we do for RA's weapons factory.

Reply:
This is the sprite body that is used for the `make` animation so that won't suffice.